docs/platforms/javascript/common/configuration/options.mdx

```diff
@@ -399,8 +399,6 @@ Set this boolean to `false` to disable the auto initialization of the native lay
 
 To disable the native layer, use [enableNative](#enableNative).
 
-You should follow the [guide to native initialization](/platforms/react-native/manual-setup/native-init/) if you chose to use this option.
```

Comment:
Note: this is part of the `<PlatformSection supported={["javascript.cordova", "javascript.capacitor"]}>` defined [above](https://github.com/getsentry/sentry-docs/pull/11193/files#diff-96c5d3a51798dece73da3ce1b4537fac346e8bc11f5f331c57bcb48e6e7efeaaR378) so only affects these two platforms